CHANGELOG.md

```diff
@@ -0,0 +1,396 @@
+# Changelog
+
+## 2.35
+2019 Nov 01
+
+- Update durable-task (v1.33)
+- Add `pwsh` step to support powershell name change ([JENKINS-48803](https://issues.jenkins-ci.org/browse/JENKINS-48803))
```

Comment:
I might reword like this:

```suggestion
- Add `pwsh` step, which works exactly like the existing `powershell` step, but runs `pwsh.exe` instead of `powershell.exe`, for use with PowerShell Core 6.0 and newer. ([JENKINS-48803](https://issues.jenkins-ci.org/browse/JENKINS-48803))
```